src/coach/heuristics/OffensiveScore.cpp

```diff
@@ -41,8 +41,8 @@ bool OffensiveScore::positionIsValid(const Field &field, const Vector2 &defaultZ
     }
 
     // check if the point is in the field and out of the defense area
-    if (!FieldComputations::pointIsInField(field, positionToCheck, Constants::ROBOT_RADIUS() * 6) ||
-        FieldComputations::pointIsInDefenceArea(field, positionToCheck, false, Constants::ROBOT_RADIUS() * 2)) {
+    if (!FieldComputations::pointIsInField(field, positionToCheck, -Constants::ROBOT_RADIUS()*6) ||
```

Comment:
So you imply that both of these functions always return False? If that is the case then an issue should be made for this. However it should not be fixed in this PR, because it is out the scope of this PR.

Reply:
I know, it's just things that i noticed